tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -159,6 +161,19 @@ When('the user uploads file {string} using the webUI', function (element) {
   return client.page.filesPage().uploadFile(element)
 })
 
+When('the user uploads a folder containing following files using the webUI:', async function (files) {
+  const uploadedFiles = []
+  for (const [file] of files.raw()) {
+    const filePath = path.join(client.globals.filesForUpload, file)
+    await client.uploadRemote(filePath, uploadedFiles.push.bind(uploadedFiles))
+  }
+  // we can extract folder path from `uploadedFiles`
+  // eg: `/tmp/545cfb7e4b6e6603b2bbef4d69f82627/upload1734400381952243704file/new-lorem.txt`
+  // extract `upload1734400381952243704file` and send it as arg to `uploadFolder()`
+  console.log('Uploaded files:', uploadedFiles)
```

Comment:
remove this debugging line

Reply:
I want to keep this, kind of like a log so that people know files are being uploaded through this mechanism. What do you think?